tests/bud.bats

```diff
@@ -6486,6 +6486,15 @@ _EOF
   done
 }
 
+@test "build must reset platform for stages if needed" {
+  otherarch="arm64"
+  # just make sure that other arch is not equivalent to host arch
+  if [[ "$otherarch" == "$myarch" ]]; then
```

Comment:
`$myarch` has not been defined at this point, so the test is always false.

Reply:
Added.